app/res/layout/choice_dialog_view.xml

```diff
@@ -25,19 +25,19 @@
         android:layout_marginEnd="@dimen/standard_spacer_large">
     </ListView>
 
-    <Button xmlns:android="http://schemas.android.com/apk/res/android"
+    <com.google.android.material.button.MaterialButton
```

Comment:
Also requesting a screenshot of the choice dialog with buttons in product description. (You can see it when configuing a [repeat group](https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143956644/Repeat+Groups) in the form)

Reply:
Here's an example from the `Exit form` dialog:
![exit_form_dialog_new_ui](https://github.com/user-attachments/assets/69392efb-963d-4043-847f-94deb3b7ee1f)
I don't think these need to be rounded corners buttons, here's the current design:
![image](https://github.com/user-attachments/assets/0911f79a-ac89-4fba-bf79-2b5c2b30fa46)